app/controlplane/internal/data/ent/schema/integration.go

```diff
@@ -40,8 +40,8 @@ func (Integration) Fields() []ent.Field {
 			Default(time.Now).
 			Immutable().
 			Annotations(&entsql.Annotation{Default: "CURRENT_TIMESTAMP"}),
-		// kind specific config
-		field.Bytes("config").GoType(&pb.IntegrationConfig{}),
+		// NOTE: Optional for now until we migrate all the integrations to this new schema
+		field.JSON("conf", &anypb.Any{}).Optional(),
```

Comment:
Creating a new column since the previous one is incompatible with the new content